src/Symfony/Component/Notifier/Bridge/Firebase/README.md

```diff
@@ -1,14 +1,13 @@
 Firebase Notifier
 =================
 
-Provides Firebase integration for Symfony Notifier.
+Provides [Firebase](https://firebase.google.com) integration for Symfony Notifier.
 
 DSN example
 -----------
 
-```
-// .env file
-FIREBASE_DSN=firebase://USERNAME:PASSWORD@default
+```ini
```

Comment:
The `ini` confuses me.

Reply:
It brings (at least in PHPStorm) a better highlighting.

Reply:
We don"t need highlightning IMHO